x-pack/plugins/infra/public/pages/metrics/inventory_view/lib/create_inventory_metric_formatter.ts

```diff
@@ -20,71 +21,71 @@ interface MetricFormatter {
   bounds?: { min: number; max: number };
 }
 
-interface MetricFormatters {
-  [key: string]: MetricFormatter;
-}
+type MetricFormatters = {
+  [key in SnapshotMetricType]?: MetricFormatter;
```

Comment:
Small improvement to strictly type `MetricFormatters` possible keys

Reply:
Nice 🙂